src/styles/_section_page_header.scss

```diff
@@ -158,81 +166,92 @@
       }
     }
   }
+  @include screen-md-min {
 
-  width: 100%;
-}
+    &.header__nav--left {
 
-.header__nav--overlay {
-  @include transition(opacity 600ms ease, visibility 600ms ease, z-index 600ms ease);
+      & > ul {
+        text-align: left;
+      }
 
-  opacity: 1;
-  visibility: visible;
-  z-index: 100;
-}
+      .dropdown,
+      .header__nav-sign-in {
+        float: right;
+      }
 
-.header__nav-left,
-.header__nav-right {
-  @include align-self(center);
-  @include screen-md-min {
-    @include make-col(6);
+      .dropdown__menu {
+        right: -10px;
+      }
+    }
   }
 
-  margin-left: auto;
+  width: 100%;
 
   & > ul {
+    text-align: right;
 
     & > li {
       @include screen-md-min {
 
-        &:last-child {
-          margin-right: 0;
+        &:first-child {
+          margin-left: 0;
         }
       }
     }
   }
 }
 
-.header__nav-primary {
-  @include screen-md-min {
-    @include make-col(9);
-  }
+.header__nav--overlay {
+  @include transition(opacity 600ms ease, visibility 600ms ease, z-index 600ms ease);
+
+  opacity: 1;
+  visibility: visible;
+  z-index: 100;
 }
 
-.header__nav-secondary {
+.header__nav-item {
   @include screen-md-min {
-    @include make-col(3);
-  }
-}
+    margin-left: 15px;
 
-.header__nav-left {
-  width: auto;
-}
+    &.dropdown {
+      margin-right: 10px;
+    }
+
+    & > a {
+      color: $nav__link-color;
+      font-weight: 600;
+      text-decoration: none;
+      text-transform: uppercase;
+
+      &:active,
+      &:focus,
+      &:hover {
+        color: $nav__link-color--active;
+      }
 
-.header__nav-right {
-  text-align: right;
+      .home_landing_page header[class="header floating"] &,
```

Comment:
why not 
```scss
.home_landing_page .header.floating ?
```

Reply:
@derek-findthebrink @leonardofaria The reason for header[class="header floating"] is because this specifies when the <header> element has ONLY these two classes. When I targeted it as a normal selector chain, it didn't behave as intended.

Reply:
ah! Interesting!